tests/integration/records/single_db/numeric_expectations.py

```diff
@@ -120,11 +120,11 @@
         'uint64': 'NUMERIC(20, 0)',
         'float16': 'REAL',
         'float32': 'REAL',
-        'float64': 'DOUBLE PRECISION',
-        'float128': 'DOUBLE PRECISION',  # Redshift doesn't support >float64
+        'float64': 'DOUBLE_PRECISION',
+        'float128': 'DOUBLE_PRECISION',  # Redshift doesn't support >float64
```

Comment:
Why did this change as a result of upgrading airflow? were float128s ever actually used for redshift?